examples/cross.js

```diff
@@ -0,0 +1,69 @@
+var data = require('./data/data.json')
+var streamSift = require('../lib')
+var c = require('colors/safe')
+
+/**
+* $cross
+* $crossOrEqual
+*/
+
+var spec = { input : { $c : 1 } }
+var ss = streamSift(spec)
+
+data.map(function(obj){
```

Comment:
To be clear, one that will return a function suitable for `data.map` to consume.